packages/logging/src/servicies/logger/logger.service.ts

```
@@ -83,6 +95,10 @@ export class LoggerService implements Logger {
     return this;
   }
 
+  log(message: any): void {
+    return this.pino.info(message);
+  }
+
   info(message: any): void {
     return this.pino.info(this.toLoggableInput(message));
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)